var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -30,6 +30,8 @@ class Libxc(AutotoolsPackage, CudaPackage):
     version("2.2.1", sha256="ade61c1fa4ed238edd56408fd8ee6c2e305a3d5753e160017e2a71817c98fd00")
 
     variant("shared", default=True, description="Build shared libraries")
+    variant("kxc",default=False, description="Build with third derivatives")
+    variant("lxc",default=False, description="Build with fourth derivatives")
```

Comment:
Do you want to recognize that these options were added in `5.0.0` using a conditional variant (i.e., adding the `when` argument)?

(See https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants.)